strscan.gemspec

```
@@ -19,14 +19,14 @@ Gem::Specification.new do |s|
   files = [
     "COPYING",
     "LICENSE.txt",
-  ]
+  ] + Dir[File.join(__dir__, "lib/**/*.rb")]
```

Could you use a static list? Glob may not work in ruby/ruby.

```suggestion
    "lib/strscan.rb",
  ]
```

---

Done.